instrumentor/controllers/instrumentationdevice/common.go

```diff
@@ -133,7 +133,7 @@ func addInstrumentationDeviceToWorkload(ctx context.Context, kubeClient client.C
 			agentsCanRunConcurrently = *odigosConfiguration.AllowConcurrentAgents
 		}
 
-		err, deviceApplied, deviceSkippedDueToOtherAgent := instrumentation.ApplyInstrumentationDevicesToPodTemplate(podSpec, runtimeDetails, otelSdkToUse, obj, logger, agentsCanRunConcurrently)
+		err, deviceApplied, deviceSkippedDueToOtherAgent := instrumentation.ApplyInstrumentationDevicesToPodTemplate(podSpec, instApp.Spec.RuntimeDetails, otelSdkToUse, obj, logger, agentsCanRunConcurrently)
```

Comment:
Should we use the instru. config instead?
`instruConfig.Status.RuntimeDetailsByContainer`

Reply:
I'll rename it in the next PR after the passed argument will be of type instConfig